### PR TITLE
fix: OpenApi Parser v2 return auths if types are nullish

### PR DESCRIPTION
**Pull request by RohinBhargava**
## Short description of the changes made

* Previously, auths were not propagating if there were no schemas. Now they do

## What was the motivation & context behind this PR?

* Customer edge case

## How has this PR been tested?

* Snapshot test

### CODE REVIEW

File: packages/parsers/src/openapi/3.1/OpenApiDocumentConverter.node.ts [hunk 1]
```diff
@@ -138,6 +138,7 @@ export class OpenApiDocumentConverterNode extends BaseOpenApiV3_1ConverterNode<
     > = computeSubpackages({ endpoints, webhookEndpoints: webhooks });
 
     const { types, auths } = this.components?.convert() ?? {};
+    console.log(types, auths);
```
Comment (Swimburger) [inline packages/parsers/src/openapi/3.1/OpenApiDocumentConverter.node.ts:141]: intentional log?

**Review by Swimburger — APPROVED** [review]
